kube/client.go

```diff
@@ -48,20 +52,18 @@ var pruneWhitelist = []string{
 	"networking.k8s.io/v1/NetworkPolicy",
 }
 
-// AutomaticDeploymentOption type used for labels
-type AutomaticDeploymentOption string
-
-// Automatic Deployment labels
-const (
-	DryRun AutomaticDeploymentOption = "dry-run"
-	On     AutomaticDeploymentOption = "on"
-	Off    AutomaticDeploymentOption = "off"
-)
+// KAAnnotations contains the standard set of annotations on the Namespace
+// resource defining behaviour for that Namespace
+type KAAnnotations struct {
+	Enabled string
```

Comment:
How about having these as bool and move out the string to bool logic outside of `Apply`?

Reply:
The reason is there, I couldn't think of a nice way to communicate optionals.

False because its set to false, or because its absent, I didn't want that logic in the client, since all the "behavior" logic is here, in the BatchApplier.